extensions/hide-files/src/unhide-all-files.tsx

```diff
@@ -0,0 +1,14 @@
+import { showHUD } from "@raycast/api";
+import { exec } from "child_process";
+import { getFocusFinderPath } from "./utils/common-utils";
+import { removeFilesFromPanel } from "./utils/hide-files-utils";
+
+export default async () => {
+  await showHUD("Unhidden Current window files");
+  const finderPath = await getFocusFinderPath();
+  const hideDesktopFilesCommand = `chflags nohidden ${finderPath.replace(" ", `" "`)}*`;
+  exec(hideDesktopFilesCommand);
```

Comment:
hi @koinzhang - what about instead of using `spawn` instead of `exec`? In general we try not to use `exec` at all if possible; using it in combination with string replacement sounds a bit risky…

Reply:
Hi @sxn. Thank you very much for the advice, I learned something new.
I have replaced all `execs` with `spawn`.